streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverSchedulingPolicy.scala

```diff
@@ -30,22 +30,22 @@ import org.apache.spark.util.Utils
  * - The first phase is global scheduling when ReceiverTracker is starting and we need to schedule
  *   all receivers at the same time. ReceiverTracker will call `scheduleReceivers` at this phase.
  *   It will try to schedule receivers with evenly distributed. ReceiverTracker should update its
```

Comment:
nit: grammar. It will try to scheduler receiver such that they are evenly distributed